src/components/views/AgreementLine.js

```diff
@@ -66,59 +71,81 @@ const AgreementLine = ({
-          <Button
-            buttonStyle="dropdownItem"
-            id="clickable-dropdown-edit-agreement-line"
-            onClick={handlers.onEdit}
-          >
-            <Icon icon="edit">
-              <FormattedMessage id="ui-agreements.agreements.edit" />
-            </Icon>
-          </Button>
-          <Button
-            buttonStyle="dropdownItem"
-            id="clickable-dropdown-delete-agreement-line"
-            onClick={handlers.onDelete}
-          >
-            <Icon icon="trash">
-              <FormattedMessage id="ui-agreements.delete" />
-            </Icon>
-          </Button>
-        </IfPermission>
-      )}
-      appIcon={<AppIcon app="agreements" />}
-      paneTitle={<FormattedMessage id="ui-agreements.agreementLine" />}
-      {...paneProps}
-    >
-      <Info line={line} resource={resource} />
-      <AccordionStatus>
-        <Row end="xs">
-          <Col xs>
-            <ExpandAllButton id="clickable-expand-all" />
-          </Col>
-        </Row>
-        <AccordionSet>
-          <POLines line={line} resource={resource} />
-          <Coverage line={line} resource={resource} />
-          <FormattedMessage id="ui-agreements.line.lineForAgreement" values={{ agreementName: line.owner?.name }}>
-            {title => (
-              <NotesSmartAccordion
-                domainName="agreements"
-                entityId={line.id ?? '-'}
-                entityName={title ?? '-'}
-                entityType="agreementLine"
-                id="agreement-line-notes"
-                pathToNoteCreate={urls.noteCreate()}
-                pathToNoteDetails={urls.notes()}
-              />
-            )}
-          </FormattedMessage>
-        </AccordionSet>
-      </AccordionStatus>
-    </Pane>
+    <>
+      <Pane
+        actionMenu={() => (
+          <IfPermission perm="ui-agreements.agreements.edit">
+            <Button
+              buttonStyle="dropdownItem"
+              id="clickable-dropdown-edit-agreement-line"
+              onClick={handlers.onEdit}
+            >
+              <Icon icon="edit">
+                <FormattedMessage id="ui-agreements.agreements.edit" />
+              </Icon>
+            </Button>
+            <Button
+              buttonStyle="dropdownItem"
+              id="clickable-dropdown-delete-agreement-line"
+              onClick={handlers.onDelete}
+            >
+              <Icon icon="trash">
+                <FormattedMessage id="ui-agreements.delete" />
+              </Icon>
+            </Button>
+          </IfPermission>
+        )}
+        appIcon={<AppIcon app="agreements" />}
+        lastMenu={
+          <IfPermission perm="ui-agreements.agreements.edit">
+            <PaneMenu>
+              {handlers.onToggleTags &&
+                <FormattedMessage id="ui-agreements.agreements.showTags">
+                  {ariaLabel => (
+                    <IconButton
+                      ariaLabel={ariaLabel}
```

Comment:
Can just do `useIntl` and `formatMessage` now instead of the render-props form of `FormattedMessage`. 👍